packages/datepicker/src/ux-datepicker.ts

```diff
@@ -50,6 +59,10 @@ export class UxDatepicker implements UxComponent {
   public bind() {
     this.processAttribute('placeholder');
 
+    if (this.autofocus || this.autofocus === '') {
+      this.focused = true;
```

Comment:
when setting this and the element is not yet attached, will it actually be focused as expected by `autofocus`?

Reply:
Yes it works. At least from my test in browser (chrome). Again, I have mimic the behavior of `ux-input`.